objcgen/objcgenerator-processor.cs

```diff
@@ -196,6 +202,13 @@ public override void Process (IEnumerable<Assembly> assemblies)
 					if (props.Count > 0)
 						properties.Add (t, props);
 
+					if (subscriptProps.Count > 0) {
+						if (subscriptProps.Count > 1)
+							delayed.Add (ErrorHelper.CreateWarning (1041, $"Indexed properties on {t.Name} is not generated because multiple indexed properties not supported."));
```

Comment:
You're just showing the warning here, but not changing any behavior, which means the properties will still be generated.